### PR TITLE
[TS-115] cypress testing for dashboard visibility

### PR DESCRIPTION
**Pull request by Ming424**
Finish cypress testing for dashboard components

image

### CODE REVIEW

**Comment by itsShaVawn** [comment]
@Ming424 Can you change the PR message and the commit message to follow our naming guidelines please 🙂

File: cypress/integration/E2E_tests/DashboardComponentsTests/holdings.spec.js [hunk 1]
```diff
@@ -8,12 +8,50 @@ describe("On the dashbooard page, Holdings component can", () => {
...
+      .clear()
+      .type('10')
+      .should('have.value', '10');
+    cy.get('[data-cy=positions-new-item-pl-tf]')
+      .clear()
+      .type('123')
+      .should('have.value', '123');
+    cy.get('[data-cy=positions-new-item-submit-btn]').click();
+  });
+  it("display recvent trades component", () => {
```
Comment (AlyaNaseer) [inline cypress/integration/E2E_tests/DashboardComponentsTests/holdings.spec.js:38]: You've got a typo here ("recvent")

**Comment by AlyaNaseer** [comment]
I'm not sure if we should be testing for the specific values within the component, but rather just the visibility, since that number will change once we fetch the data from the API (and we don't want to have to go back and rework all our Cypress tests)

File: cypress/integration/E2E_tests/DashboardComponentsTests/holdings.spec.js (hunk 1 above)
Comment (Ming424) [reply cypress/integration/E2E_tests/DashboardComponentsTests/holdings.spec.js]: edited

**Comment by AlyaNaseer** [comment]
Looks good! Just need to merge main into then I'll approve (if I approve now, it'll dismiss it as a stale review)

**Comment by Ming424** [comment]
@AlyaNaseer merged main into TS-155

**Review by AlyaNaseer — APPROVED** [review]
Works locally! Just change the title of the PR to TS-155 instead of TS-115 :)

**Review by kennguyen0303 — APPROVED** [review]

**Pull request merged**